onnxruntime/contrib_ops/cuda/moe/ft_moe/cutlass_heuristic.h

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2020-2023, NVIDIA CORPORATION.  All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include "ft_gemm_configs.h"
+
+#include <cstddef>
+#include <cstdint>
+#include <vector>
+
+// #include "src/fastertransformer/utils/cuda_utils.h"
+
+namespace fastertransformer {
```

Comment:
suggest to change namespace. Either
```
namspace onnxruntime{
namespace fastertransformer {
```
Or
```
namespace ort_fastertransformer
```

Just in case name conflict when someone build an application with both ORT and FasterTransformer.